container/container_windows.go

```diff
@@ -47,14 +49,44 @@ func (container *Container) IpcMounts() []Mount {
 	return nil
 }
 
-// SecretMounts returns the mounts for the secret path
-func (container *Container) SecretMounts() []Mount {
+// CreateSecretSymlinks creates symlinks to the secret volume.
+func (container *Container) CreateSecretSymlinks() error {
+	for _, r := range container.SecretReferences {
+		if r.File == nil {
```

Comment:
IIRC `File` is in a `oneof` in the swarmkit gRPC API. If we add other possible types of secret targets, to allow exposing them in other ways than as files, `File` will be nil.